bodyprocessors/multipart.go

```diff
@@ -50,18 +51,29 @@ func (mbp *multipartBodyProcessor) ProcessRequest(reader io.Reader, collections
 		// if is a file
 		filename := originFileName(p)
 		if filename != "" {
-			temp, err := os.CreateTemp(storagePath, "crzmp*")
-			if err != nil {
-				return err
-			}
-			sz, err := io.Copy(temp, p)
-			if err != nil {
-				return err
+			var size int64
+			if !environment.IsTinyGo {
+				// Only copy file to temp when not running in TinyGo
+				temp, err := os.CreateTemp(storagePath, "crzmp*")
+				if err != nil {
+					return err
+				}
+				sz, err := io.Copy(temp, p)
+				if err != nil {
+					return err
+				}
+				size = sz
+				filesTmpNamesCol.Add("", temp.Name())
```

Comment:
I'm not too familiar with the variable collections infrastructure yet, can anyone confirm it's ok to only add this conditionally, or I should add a dummy value in the tinygo path too?

Reply:
FTW gave me the answer 😂